### PR TITLE
Add Travis CI support

### PR DESCRIPTION
**Pull request by ruimarinho**

### CODE REVIEW

**Comment by nunofgs** [comment]
@ruimarinho: Tests are failing. Can you take a look?

**Comment by ruimarinho** [comment]
Yep. Fixed.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+env:
+  - VERSION=5
+  - VERSION=4
+
+language: bash
+
+script:
+  - docker build -t "$image" .
+  - docker run "$image" --version | egrep v${VERSION}\.\[0-9]\.\[0-9]
```
Comment (nunofgs) [inline .travis.yml:17]: `\d` instead of `[0-9]`?

Comment (ruimarinho) [inline .travis.yml:17]: That was the issue 😜

**Pull request merged**